nntrainer/include/activation_layer.h

```diff
@@ -76,7 +76,7 @@ class ActivationLayer : public Layer {
    *
    * @param[in] ActivationTypeeActivationTypeeActivationTypeet
    */
-  void setActivation(ActivationType acti_type);
+  int setActivation(ActivationType acti_type);
```

Comment:
instead of changing the return type of this to int, change the signature of `setActivaiton` in `layer.h` to void?

Reply:
Sure I'll do that

Reply:
Done